1.9/api/package-manager.yaml

```diff
@@ -0,0 +1,1979 @@
+                      maintainer: support@mesosphere.io
+                      description: Apache Cassandra
+                      tags:
+                        - database
+  /service/start:
+    post:
+      operationId: POST_service-start
+      description: |
+        **Experimental.** Starts a DC/OS Service from a DC/OS package.
+      consumes:
+        - application/vnd.dcos.sevice.start-request+json;charset=utf-8;version=v1
+      produces:
+        - application/vnd.dcos.sevice.start-response+json;charset=utf-8;version=v1
+      parameters:
+        - name: body
+          in: body
+          schema:
+            example:
+              packageName: cassandra
+              packageVersion: 1.0.24-3.0.10
+            $ref: |
+              #/definitions/{
+                "$schema": "http://json-schema.org/draft-04/schema#",
+                "type": "object",
+                "properties": {
+                  "packageName": {
+                    "type": "string"
+                  },
+                  "packageVersion": {
+                    "type": "string"
+                  },
+                  "options": {
+                    "type": "object",
+                    "additionalProperties": true
+                  }
+                },
+                "required": [
+                  "packageName"
+                ],
+                "additionalProperties": false
+              }
+        - name: Accept
+          in: header
+          required: false
+          enum:
+            - application/vnd.dcos.sevice.start-response+json;charset=utf-8;version=v1
+          type: string
+      responses:
+        '200':
+          description: ''
+          schema:
+            $ref: |
+              #/definitions/{
+                "$schema": "http://json-schema.org/draft-04/schema#",
+                "type": "object",
+                "properties": {
+                  "packageName": {
+                    "type": "string"
+                  },
+                  "packageVersion": {
+                    "type": "string"
+                  },
+                  "appId": {
+                    "type": "string"
+                  }
+                },
+                "required": [
+                  "packageName",
+                  "packageVersion"
+                ],
+                "additionalProperties": false
+              }
+          examples:
+            application/vnd.dcos.sevice.start-response+json;charset=utf-8;version=v1:
+              packageName: cassandra
+              packageVersion: 1.0.24-3.0.10
+              appId: /cassandra
+  /package/add:
+    post:
+      operationId: POST_package-add
+      description: |
+        **Experimental.** Adds a DC/OS Package to DC/OS. To add a DC/OS package from Universe use
+        the content type _application/vnd.dcos.package.add-request+json;charset=utf-8;version=v1_.
+        To add a package by uploading the file use the content type
```

Comment:
This swagger file is generated by the cosmos CI. The raml source is here: https://github.com/dcos/cosmos/blob/master/cosmos-server/src/main/resources/com/mesosphere/cosmos/handler/api.raml